x/auth/tx/gogotx.go

```diff
@@ -31,6 +31,9 @@ import (
 func newWrapperFromDecodedTx(
 	addrCodec address.Codec, cdc codec.BinaryCodec, decodedTx *decode.DecodedTx,
 ) (*gogoTxWrapper, error) {
+	if decodedTx.Tx.AuthInfo.Fee == nil {
```

Comment:
I think the intended behavior of this function is to not error (or panic) on nil `Fee`.  We should instead just let the empty `fees` variable fall through if `Fee` is nil on the `AuthInfo`